### DISCUSSION TITLE
When can we have chat-with-pdf option?

### DISCUSSION THREAD

**Discussion by OpinionMatter** [Mobile-Artificial-Intelligence/maid#635] [Ideas]
Many users have suggested this feature, when is it expected to be available?

**Comment by y-ziane** [comment]
+1

**Reply by Disonantemus** [reply]
Smartphone RAM is not enough to use a very big model that can do that, maybe doing 2 steps:

1. Using a model for embedding to do the OCR and create a markdown with extracted information, then ...

2. Run another model using previous extracted information to answer.

Something like a poor man's RAG, asking to just one file.
